server/api/lineitems.js

```diff
@@ -1,8 +1,30 @@
 const router = require('express').Router();
+const LineItem = require('../db/LineItem');
+const Order = require('../db/Order');
+const User = require('../db/User');
+const Product = require('../db/Product');
+
 module.exports = router;
 
-router.get('/', (req, res) => {
+router.get('/', (req, res, next) => {
+  return LineItem.findAll({
+    include: [{ all: true }]
+  })
+    .then(lineitems => res.send(lineitems));
+});
+
+router.get('/:userId/:orderId', (req, res, next) => {
```

Comment:
We don't need this get route. 
we're probably are going to do ...

Order.findById(id, { include : [{ Models : LineItems}])

Which will give us the order and the line items that come with is. We wont ever need the lineitems independently.

Reply:
I am guessing Annie wanted the product info that associates with the lineitem?
If so, as I mentioned above, we should do a nested include when we retrieve order.